File: lab6/app/app/src/main/java/com/example/myapp/data/dto/RouteLinerDto.java
package com.example.myapp.data.dto;

import java.math.BigDecimal;

public class RouteLinerDto {
    public int routeId;
    public int linerId;
    public String season;
    public double basePrice;

    public RouteLinerDto(int routeId, int linerId, String season, double basePrice) {
        this.routeId = routeId;
        this.linerId = linerId;
        this.season = season;
        this.basePrice = basePrice;
    }
}

File: lab6/TripAPI/TripAPI/DTOs/RouteLinerDto.cs
namespace TripAPI.DTOs
{
    public class RouteLinerDto
    {
        public int RouteId { get; set; }
        public string RouteName { get; set; } = null!;
        public int LinerId { get; set; }
        public string LinerName { get; set; } = null!;
        public string? Season { get; set; }
        public decimal? BasePrice { get; set; }
    }
}
